### ISSUE TITLE
Please have support for Linux/GNU Konsole/Terminal and Windows CMD ❗ ❗ ❗ 

### ISSUE THREAD

**Issue by MiChaelinzo** [LazoCoder/Pokemon-Terminal#192]
```
#pokemon pikachu
Your terminal emulator isn't supported at this time.

#pokemon -ss
Starting slideshow with 768 Pokemons and a delay of 10.0 minutes.
Forked process to background with PID 20336.
You can stop it with 'pokemon -c'.
Your terminal emulator isn't supported at this time.

```

Microsoft Windows [Version 10.0.19041.508] & Linux 5.8.0-amd64 #1 SMP Debian 5.8.10 (2020-09-22) x86_64 GNU/Linux

Works really well imo  ❗ ❗ ❗ 
![pokemon1](https://user-images.githubusercontent.com/68110223/94937202-f2a03f00-04d7-11eb-82eb-842f9b04626b.GIF)

**Comment by MiChaelinzo** [comment]
![pokemon2](https://user-images.githubusercontent.com/68110223/94937500-5591d600-04d8-11eb-838b-9bd979768a91.GIF)

**Comment by sylveon** [comment]
Windows Terminal is already supported, and the issue for Konsole is #181. Are you having any problem with the WT integration?

**Comment by MiChaelinzo** [comment]
> Windows Terminal is already supported, and the issue for Konsole is #181. Are you having any problem with the WT integration?

Yes for CMD & Powershell but downloaded WT in MS I'm not. So how do ifix konsole & xfce, mate terminal etc. for linux/gnu?
I need to download  tilix and terminology user it there instead ?
